src/fontra/views/editor/panel-reference-font.js

```diff
@@ -322,6 +323,43 @@ export default class ReferenceFontPanel extends Panel {
         cssString;
       this.editorController.canvasController.requestUpdate();
     });
+
+    this.initActions();
+  }
+
+  initActions() {
+    const topic = "0200-action-topics.reference-font";
+    registerAction(
+      "action.select-previous-reference-font",
+      {
+        topic,
+        titleKey: "menubar.view.select-previous-reference-font",
```

Comment:
These keys are no longer correct.

How about
- "reference-font.select-previous-reference-font"
- "reference-font.select-next-reference-font"

?

Reply:
Thanks. I update the titleKeys in the .js file, as well as in the google sheet and I also updated the languages files after that.